server/controllers/database-controller.js

```diff
@@ -64,30 +65,30 @@ timepointsSources.forEach(source => timepointsByDataset[source.key] = source.val
 
 let buildTimepointsQuery = function (selection) {
     let timepoints = "";
-    selection.forEach(x => timepoints += ("timepoints=\'" + x + "\' OR "));
+    selection.forEach(x => timepoints += ("fall2021.expression.time_point=" + x + " OR "));
```

Comment:
This string concat could be more consistent by following same style as on line 75 below